lib/common.sh

```diff
@@ -84,6 +84,8 @@ if [ ! -f "${SSH_KEY}" ]; then
   mkdir -p "$(dirname "$SSH_KEY")"
   ssh-keygen -f "${SSH_KEY}" -P ""
 fi
+SSH_PUB_KEY_CONTENT="$(cat "${SSH_PUB_KEY}")"
+export SSH_PUB_KEY_CONTENT
 
```

Comment:
Why do not use the `SSH_PUB_KEY` var that already exported then creating a new var?

Reply:
`SSH_PUB_KEY` is the path to the file, but `SSH_PUB_KEY_CONTENT` is what is inside the file. (Ironic wants the content, not the path, so it can inject it directly)

Reply:
what about `IRONIC_RAMDISK_SSH_KEY="$(cat "${SSH_PUB_KEY}")"` in the `ironic_bmo_configmap.env`

Reply:
Sure, that is fine by me. I guess I was thinking that we may need `SSH_PUB_KEY_CONTENT` somewhere else also at some point.
Do you want me to change it?

Reply:
well if we have some reasons for an extra var that is fine! I just thought less vars simplify the code

Reply:
Ok, I'll let it be like it is then. The variable is anyway not exposed to the user directly (set directly internally)